src/Raven.Client/Constants.cs

```diff
@@ -159,7 +159,7 @@ private Fields()
                     
                     public const string CustomSortFieldName = "__customSort";
 
-                    public const string DocumentIdFieldName = "__document_id";
+                    public const string DocumentIdFieldName = "id()";
 
                     public const string ReduceKeyFieldName = "__reduce_key";
```

Comment:
What about this, isn't this `key()` now?

Reply:
Hmm, I think that you are right... @arekpalinski ?
I will create a separate issue so we can review all other index fields.

Reply:
`__reduce_key` is _hash_ of the reduce key, we also have `__reduced_val` which is the actual object that we group by so it can be called `key()` now

Reply:
so let us call `__reduce_key` `hash(key())` ? :-)

Reply:
yes, however `hash(key().Values)` would be more exact :)

Reply:
so maybe `hash_key_values()` ?

Reply:
I prefer `hash(key())` - shorter and it expresses what it does under the hood pretty well

Reply:
go for it